pages/services/dcos-monitoring/1.0.0/operations/grafana/dashboard-configs/index.md

```diff
@@ -0,0 +1,149 @@
+For instance, `https://github.com/company/dashboard-configs/production`.
+
+```json
+{
+  "grafana": {
+    "dashboard_config_repository": {
+      "url": "https://github.com/company/dashboard-configs",
+      "path": "/production"
+    }
+  }
+}
+```
+
+The Git repository can be either public or private.
+If the Git repository is private, you will need to configure the credentials to access the Git repository (see below).
+
+<p class="message--note"><strong>NOTE:</strong> If enabled, the set of default dashboards shipped with the service is from the Mesosphere-maintained Git repository `https://github.com/dcos/grafana-dashboards`.
+These dashboards are updated with {{ model.techName }} service releases.
+If you would like to use the most up-to-date version of these dashboards that has not yet been released, you can configure the repository url to point to `https://github.com/dcos/grafana-dashboards`.
+The `grafana.default_dashboards` option should be set to `false`.
+As it is a public repository, there is no need to set up the `credentials`.</p>
+
+```json
+{
+  "grafana": {
+    "default_dashboards": false,
+    "dashboard_config_repository": {
+      "url": "https://github.com/dcos/grafana-dashboards",
+      "path": "/dashboards"
+    }
+  }
+}
+```
+
+## Create secrets for Git repository credentials
+
+If the Git repository containing the Grafana dashboard configurations is private, you will need to configure the secrets first.
+Currently, the following Auth types are supported.
+
+### HTTP Auth
+
+```bash
+dcos security secrets create --value=<GITHUB_USERNAME> githubusername-secret
+dcos security secrets create --value=<GITHUB_PASSWORD> githubpassword-secret
+```
+
+For Github, the password can be the API token.
+
+Create a custom option file (`options.json`) like the following.
+You can omit the `credentials` section if the Git repository is public.
+
+```json
+{
+  "grafana": {
+    "dashboard_config_repository": {
+      "url": "https://github.com/company/dashboard-configs",
+      "path": "/production",
+      "credentials": {
+        "username_secret": "githubusername-secret",
+        "password_secret": "githubpassword-secret"
+      }
+    }
+  }
+}
+```
+
+### SSH Auth
+
+```bash
+dcos security secrets create -f <PATH_TO_PRIVATE_KEY> gitsshkey-secret
+```
+
+For Github, please make sure to add [Deployment Key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) (i.e., the public key) to the repository.
+
+Create a custom option file (`options.json`) like the following.
+
+```json
+{
+  "grafana": {
+    "dashboard_config_repository": {
+      "url": "git@github.com:company/dashboard-configs.git",
+      "path": "/production",
+      "credentials": {
+        "deploy_key_secret": "gitsshkey-secret"
+      }
+    }
+  }
+}
+```
+
+Note that you'll have to use `git@github.com:<USER>/<REPO>.git` instead of `https` as the scheme of the URL.
```

Comment:
Use the `Note` message format listed on https://wiki.mesosphere.com/display/DOCS/Documentation+Style+Guide#DocumentationStyleGuide-Notes